packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -63,6 +66,8 @@ import { StyledSvgWarning, YellowText } from '../VideoWorkspace.style'
 const CUSTOM_LICENSE_CODE = 1000
 const SCROLL_TIMEOUT = 700
 const MINT_NFT_TIMEOUT = 1200
+const MIN_TITLE_LENGTH = 5
```

Comment:
Wasn't min length 5? I think we weren't meant to change this; @toiletgranny ?

Reply:
TBH I took it from figma, but you are right we didn't discuss this. @toiletgranny WDYT?